src/tags/learn-checkpoint/learn-checkpoint.style.scss

```
@@ -0,0 +1,55 @@
+    padding: 0.35rem 0.9rem;
+    border: none;
+    border-radius: 2rem;
+    background-color: var(--section-color, var(--color-blue));
+    color: var(--color-background);
+    cursor: pointer;
+    font: inherit;
+    font-size: 0.95rem;
+
+    svg {
+      height: 0.9em;
+      fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)